unitTests/ui/DateUtilsTest.ts

```diff
@@ -120,6 +120,12 @@ export function DateUtilsTest() {
       expect(result).toContain(':00');
     });
 
+    it('dateTimeToString should work with the predefinedFormat yyyy (lowercase)', () => {
```

Comment:
Is there a test with `YYYY` ? Does it work correctly ?

Reply:
YYYY is the right format for moment, I didnt think a test was necessary for it